src/renderers/webgl/WebGLBackground.d.ts

```diff
@@ -3,17 +3,17 @@ import { Color } from '../../math/Color';
 import { WebGLRenderer } from '../WebGLRenderer';
 import { WebGLState } from './WebGLState';
 import { WebGLObjects } from './WebGLObjects';
-import { WebGLRenderLists } from './WebGLRenderLists';
+import { WebGLRenderList } from './WebGLRenderLists';
 import { Scene } from '../../scenes/Scene';
 
 export class WebGLBackground {
 
 	constructor( renderer: WebGLRenderer, state: WebGLState, objects: WebGLObjects, premultipliedAlpha: any );
 
-	getClearColor(): void;
-	setClearColor( color: Color, alpha: any ): void;
-	getClearAlpha(): void;
-	setClearAlpha( alpha: any ): void;
-	render( renderList: WebGLRenderLists, scene: Scene, camera: any, forceClear: any ): void;
+	getClearColor(): Color;
+	setClearColor( color: Color, alpha: number ): void;
+	getClearAlpha(): number;
+	setClearAlpha( alpha: number ): void;
+	render( renderList: WebGLRenderList, scene: Scene, camera: any, forceClear: any ): void;
```

Comment:
While you're at it^^:
```ts
render( renderList: WebGLRenderList, scene: Scene, camera: Camera, forceClear: boolean ): void;
```

Reply:
You can also assign `Camera` as the type of `camera`. It's still `any`.

Reply:
done